src/task1.js

```diff
@@ -0,0 +1,5 @@
+(() => {
+    process.openStdin().addListener('data', data => { 
```

Comment:
Consider using a process.stdin.on event stream in this situation.
Consider creating a separate function, e.x dataHandler, and passing it to process.stdin.on as a event handler.

Reply:
Updated as per the comment